### PR TITLE
Feature/user login

### PR DESCRIPTION
**Pull request by stekug**

### CODE REVIEW

File: components/Layout.tsx [hunk 1]
```diff
@@ -11,15 +11,21 @@ export default function Layout({ children }: ChildrenLayoutProps) {
...
   return (
-    <StyledWrapper>
-      <Header onToggleNav={handleToggleNav} />
-      {showNav ? (
-        <NavigationList onToggleNav={handleToggleNav} />
-      ) : (
-        <div>{children}</div>
-      )}
-    </StyledWrapper>
+    <>
```
Comment (NikRoe) [inline components/Layout.tsx]: You don't need fragments here

File: components/NoUserRecipes.tsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+export default function NoUserRecipes() {
+  return (
+    <StyledArticle>
+      <StyledParagraph>
+        <StyledSpan>
+          Sorry :( <br />
+          You have not created any recipes yet.
+        </StyledSpan>
+      </StyledParagraph>
+      <StyledParagraph>You can create recipes here.</StyledParagraph>
```
Comment (NikRoe) [inline components/NoUserRecipes.tsx]: My first assumption when reading this was: "Oh, I can click here and directly create a recipe."

At least for me this a little bit misleading, but maybe it's just me.

File: pages/login.tsx [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import Login from '@/components/Login';
+
+export default function UserLogin() {
+  return (
+    <>
+      <Login />
+    </>
```
Comment (NikRoe) [inline pages/login.tsx]: You don't need the fragments

**Review by NikRoe — APPROVED** [review]
Hi there,

nice work!

I left some small suggestions for you to implement.

After having incorporated them, you can continue to QA without having to repost the PR.

Keep it up 🙌

File: components/NoUserRecipes.tsx (hunk 2 above)
Comment (stekug) [reply components/NoUserRecipes.tsx]: Good Point! We really forgot to add the link there :D We will do! Thanks!

File: pages/login.tsx (hunk 3 above)
Comment (stekug) [reply pages/login.tsx]: Thanks, we love fragments! (But we will delete them... :D )

File: components/Layout.tsx (hunk 1 above)
Comment (gunnarmon) [reply components/Layout.tsx]: thanks!

**Pull request merged**